packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import { DesktopWrapper, Icon, MobileDialog, MobileWrapper, Text } from '@deriv/components';
 import { isDesktop, isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import { connect } from 'Stores/connect';
+import { connect } from '../../../../Stores/connect';
```

Comment:
```suggestion
import { connect } from 'Stores/connect';
```
@henry-deriv it has to be imported as before without muliple ../ 🙂